src/app/flowtype/Tile.tsx

```diff
@@ -0,0 +1,79 @@
+'use client';
+
+import {
+  Flex,
+  FlexItem,
+  Tile,
+  Title,
+  TitleSizes,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+interface TileProps {
+  connectorType: string;
+  onSelect: (connectorType: string) => void;
+}
+
+// const Tilee: React.FC<TileProps> = ({ connectorType, onSelect }) => {
+//   return (
+//     <div
+//       style={{
+//         border: '1px solid #ccc',
+//         padding: '10px',
+//         margin: '5px',
+//         cursor: 'pointer',
+//       }}
+//       onClick={() => onSelect(connectorType)}
+//     >
+//       {connectorType}
+//     </div>
+//   );
+// };
+
+const Tilee: React.FunctionComponent<TileProps> = ({
+  connectorType,
+  onSelect,
+}) => (
+  <div
+    role="listbox"
+    aria-label="Stacked tiles"
+    onClick={() => onSelect(connectorType)}
+  >
+    <Tile title={connectorType} isSelected>
```

Comment:
Perhaps instead of `Tile` we could use the `ToggleGroup` from Patternfly: https://patternfly-react.surge.sh/components/toggle-group

And then you won't have to worry about the CSS for the outside selection of the item, just make sure the state is correct with `isSelected` and it will highlight it. Then we can work on the CSS for the inside part with the arrow.